public/apidocs/swagger_v3.json

```diff
@@ -2076,6 +2161,49 @@
         }
       }
     },
+    "vaccineCoverage":{
+       "type":"object",
+       "properties":{
+         "date":{
+           "type":"number"
+         }
+       }
+    },
+    "vaccineCountriesCoverage":{
+       "type":"array",
+       "items":{
+         "type":"object",
+         "properties":{
+           "country":{
+             "type":"string"
+           },
+           "timeline":{
+             "type":"object",
+             "properties":{
+               "date":{
+                 "type":"number"
+               }
+             }
+           }
+         }
+       }
+    },
+    "vaccineCountryCoverage":{
+       "type":"object",
+       "properties":{
+         "country":{
+           "type":"string"
+         },
+         "timeline":{
+           "type":"object",
+           "properties":{
+             "date":{
+               "type":"number"
+             }
+           }
+         }
+       }
+    },
```

Comment:
This all looks good, just make sure you tested the swagger docs locally and they look good, because sometimes the smallest unnoticeable thing breaks them

Reply:
I have tested it locally.